src/arrayMethodSort.js

```diff
@@ -4,8 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = defaultCompareFunction) {
```
You can use default parameters https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters